eventmesh-meta/eventmesh-meta-nacos/src/main/java/org/apache/eventmesh/meta/nacos/service/NacosMetaService.java

```diff
@@ -216,13 +216,25 @@ public void shutdown() throws MetaException {
         if (!startStatus.compareAndSet(true, false)) {
             return;
         }
+        shutdownNacosService();
```

Comment:
I think this method can be inlined. The `shutdown()` interface is used to shutdown Nacos, and there is no need to extract a separate method specifically for closing Nacos.

Reply:
please review again, thanks.